tests/functional/object/mcg/test_sts_bucket.py

```
@@ -134,3 +135,117 @@ def test_sts_bucket_ops(
+            {
+                "interface": "CLI",
+                "namespace_policy_dict": {
+                    "type": "Single",
+                    "namespacestores": [namespacestores[0]],
+                },
+            },
+        ]
+        bucketnames = [
+            bucket_factory(bucketclass=bc)[0].name for bc in bucketclass_dicts
+        ]
+        logger.info(f"Created buckets: {bucketnames}")
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)